src/libraries/System.Net.WebSockets/src/System/Net/WebSockets/Compression/WebSocketInflater.cs

```diff
@@ -210,12 +211,12 @@ private void ReleaseBuffer()
             }
         }
 
-        private static unsafe bool IsFinished(ZLibStreamHandle stream, out byte? remainingByte)
+        private static bool IsFinished(ZLibStreamHandle stream, out byte? remainingByte)
         {
             // There is no other way to make sure that we've consumed all data
             // but to try to inflate again with at least one byte of output buffer.
-            byte b;
-            if (Inflate(stream, new Span<byte>(&b, 1), FlushCode.SyncFlush) == 0)
+            byte b = 0;
+            if (Inflate(stream, MemoryMarshal.CreateSpan(ref b, 1), FlushCode.SyncFlush) == 0)
```

Comment:
This should just be `new Span(ref b) `

Reply:
Right, Thanks! https://github.com/dotnet/runtime/pull/101953/commits/4708726602b0a364583010d4914440840531da23